lib/train/platforms/detect/helpers/os_linux.rb

```diff
@@ -10,7 +10,7 @@ def redhatish_version(conf)
       case conf
       when /rawhide/i
         /((\d+) \(Rawhide\))/i.match(conf)[1].downcase
-      when /Amazon Linux AMI/i
+      when /Amazon Linux (?:AMI|.*)/i
```

Comment:
Could this just be `/Amazon Linux/i` ?

Reply:
I'm not sure...I don't see why not though. I'll change it.